### PR TITLE
Makefile and makebumpver improvement

### PR DESCRIPTION
**Pull request by M4rtinK**
A couple of makefile improvements for the separate release commit workflow + release commit support for the makebumpver script.

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -75,6 +75,13 @@ po-pull:
 	rpm -q zanata-python-client &>/dev/null || ( echo "need to run: dnf install zanata-python-client"; exit 1 )
 	( cd $(srcdir) && zanata pull $(ZANATA_PULL_ARGS) )
 
+po-push:
+	$(MAKE) -C po $(PACKAGE_NAME).pot-update
+	zanata push $(ZANATA_PUSH_ARGS)
+
+po-all:
+	$(MAKE) po-push && $(MAKE) po-pull
```
Comment (vpodzime) [inline Makefile.am]: Might be better to put these commands on separate lines.

File: Makefile.am [hunk 2]
```diff
@@ -97,6 +104,9 @@ scratch-bumpver:
 	$(MAKE) -C po $(PACKAGE_NAME).pot-update
 
 release:
+	$(MAKE) po-all && $(MAKE) dist
```
Comment (vpodzime) [inline Makefile.am]: same here

File: Makefile.am [hunk 3]
```diff
@@ -97,6 +104,9 @@ scratch-bumpver:
...
+
+release-and-tag:
 	$(MAKE) dist && $(MAKE) tag
```
Comment (vpodzime) [inline Makefile.am]: and here

**Comment by vpodzime** [comment]
Looks good to me otherwise.

**Comment by M4rtinK** [comment]
@vpodzime Good point - moved those bits to separate lines. :)

**Pull request merged**